src/platform/user/authentication/utilities.js

```diff
@@ -86,6 +86,10 @@ export function login(policy, version = 'v0', application = null) {
   );
 }
 
+export function autoLogin() {
+  return redirect(sessionTypeUrl('idme', 'v1'), 'sso-automatic-login');
```

Comment:
@hinzed1127 note that this API call will return a 500 until this PR is merged https://github.com/department-of-veterans-affairs/vets-api/pull/4168

Reply:
that's a good callout, should be sufficient here as an AC that this merges in conjunction with or after that PR